Nightscout/app.json

```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "b0193185a6e5fe70dd102f701db350fbd0e79aa5",
+    "name": "Nightscout",
+    "website": "https://www.nightscout.info",
```

Comment:
Please update the url to https://nightscout.github.io/ as https://www.nightscout.info has a self signed cert.
![Screenshot 2022-12-28 at 23 28 22](https://user-images.githubusercontent.com/439392/209880865-7b4a047a-0fc3-4c5b-9eb1-91b9567b46c0.png)